src/main/resources/assets/mappet/lang/ru_RU.lang

```diff
@@ -493,6 +499,13 @@ mappet.gui.npcs.context.rename=Переназовите состояние...
 mappet.gui.npcs.context.remove=Удалить состояние
 mappet.gui.npcs.info.empty=Чтобы начать редактирование, создайте или выберите NPC из списка справа...
 
+mappet.gui.npcs.respawn.title = Возрождение
```

Comment:
There should not be spaces between key, = and language string. Also camelCase isn’t appropriate for language files, it should be under_score.